www/includes/easyparliament/member.php

```diff
@@ -21,7 +21,7 @@ class MEMBER {
     public $extra_info = array();
     // Is this MP THEUSERS's MP?
     public $the_users_mp = false;
-    public $house_disp = 0; # Which house we should display this person in
+    public $house_disp = HOUSE_TYPE_ROYAL; # Which house we should display this person in
```

Comment:
Whilst I see the reason for this, this is actually an undefined value, rather than the royal house constant. It should probably not have a default here.

Reply:
I'll revert this change back to setting house_disp to zero. I'm not confident to set it to undefined until I add unit tests, I'd better not introduce null pointer exceptions :)